checker/manual/creating-a-checker.tex

```diff
@@ -1094,31 +1087,81 @@ \section{The checker class:  Compiler interface\label{writing-compiler-interface
 checker classes for the Interning Checker and the Nullness Checker:
 
 \begin{Verbatim}
-  @TypeQualifiers({ Interned.class, PolyInterned.class, PolyAll.class })
   @SupportedLintOptions({"dotequals"})
   public final class InterningChecker extends BaseTypeChecker { }
 
-  @TypeQualifiers({ Nullable.class, Raw.class, NonNull.class, PolyNull.class, PolyAll.class })
   @SupportedLintOptions({"flow", "cast", "cast:redundant"})
   public class NullnessChecker extends BaseTypeChecker { }
 \end{Verbatim}
 
-
 The checker class must indicate the annotations
 that make up the type hierarchy for this checker (including
-polymorphic qualifiers), either via a
-\refqualclass{framework/qual}{TypeQualifiers} annotation or by overriding
+polymorphic qualifiers), either by placing the annotations inside a subfolder called \<qual> located in the same directory as the checker, or by overriding
 the
 \refmethodterse{framework/type}{AnnotatedTypeFactory}{createSupportedTypeQualifiers}{--}
-method.  Each argument to \<@TypeQualifiers> or value returned by
-\<createSupportedTypeQualifiers> is a class literal for a type qualifier
-whose definition bears the
-\refqualclass{framework/qual}{TypeQualifier} meta-annotation.
+method. By default, the Polymorphic Qualifier \refqualclass{framework/qual}{PolyAll} is not utilized, but can be optionally added to a checker.
```

Comment:
Is there any harm in always adding PolyAll?
It's much more overhead to add a qualifier without TypeQualifiers annotations. So automatically adding PolyAll seems like a good change.